docs/src/main/sphinx/release/release-463.md

```diff
@@ -12,6 +12,14 @@
   {func}`dense_rank`, {func}`percent_rank`,  {func}`cume_dist`, and
   {func}`row_number`. ({issue}`23742`)
 
+## JDBC driver
+
+* Use HTTP/2 by default. ({issue}`21793`)
```

Comment:
if this is supported by the server (or a load balancer between the client and the server). I'd just say "Support connecting over HTTP/2"

Reply:
ok

Reply:
Updated and also added to General